### PR TITLE
SEO-174767-Xamarin-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello, 

I have worked on the missing H1.

Regards,
Beryl.

### CODE REVIEW

File: Xamarin/Maps/How-To/Load-stream-to-ShapeFileLayer.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ control: SfMaps
 documentation: ug
 ---
 
-## Load map shapefiles as a stream
+# Load map shapefiles as a stream
```
Comment (Yvone-Atieno) [inline Xamarin/Maps/How-To/Load-stream-to-ShapeFileLayer.md:10]: Include platform name

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Parallax-View/custom-controls.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ control: SfParallaxView
 documentation: ug
 ---
 
-## Scrolling support for custom controls
+# Custom controls in Xamarin Parallax View Control
```
[the remark this answers is not in the document]
Comment (berylouma) [reply Xamarin/Parallax-View/custom-controls.md]: Updated

File: Xamarin/Maps/How-To/Load-stream-to-ShapeFileLayer.md (hunk 1 above)
Comment (berylouma) [reply Xamarin/Maps/How-To/Load-stream-to-ShapeFileLayer.md]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**